src/databricks/sql/thrift_backend.py

```diff
@@ -130,6 +130,10 @@ def __init__(
         # _enable_v3_retries
         # Whether to use the DatabricksRetryPolicy implemented in urllib3
         # (defaults to False)
+        # _retry_max_redirects
+        #  An integer representing the maximum number of redirects to follow for a request.
```

Comment:
Is this validated anywhere?  Specifically, what is the user experience if retry_max_redirects > retry_stop_after_attempts_count?

Reply:
I'm not validating it as far as the user is concerned. It won't raise an exception or a log message. The behaviour is validated by tests, however.

Reply:
I don't love that the user gets no message about why we didn't respect their config (and that's the way it seems to me as a user).  I think its worth logging a warning, since setting it that way implies the user doesn't understand how the configs interact.

Reply:
It's not that we don't respect the config. We do, it's just that the configs disagree. I'll add a warning for this situation.

Reply:
Thank you, I meant that it *appeared* that you didn't respect the config :).  And adding the warning can clear up that appearance.

Reply:
Added and tested in 3c0c73e28d94b81a821ac4ee5237d2828a53b525